src/types/config.types.ts

```diff
@@ -13,3 +13,5 @@ export interface ConfigKeys {
   readonly DEV_NEW_AUTH_TOKEN: string;
   readonly DEV_MNEMONIC: string;
 }
+
+export const CREDENTIALS_FILE = './dist/cli.inxt';
```

Comment:
This is probably going to give us problems when bundling the CLI, I suggest you to do something like:

`export const CREDENTIALS_FILE = path.join(process.cwd(), '/dist/cli.inxt')`

Reply:
here is fine, as long as we use absolute paths we are safe